docs/modules/core/pages/slide.adoc

```diff
@@ -132,6 +132,11 @@ fun Title() {
 Bullet points are a standard way of presenting a list of items in presentations.
 CuP provides a `BulletPoints` composable function that animates each item as it appears:
 
+Currently, bullet points *are not* part of the core of *CuP*.
```

Comment:
Remove "Currently".
Bullet points won't ever be put back to core. That's the point of this PR.